packages/terra-notification-dialog/src/terra-dev-site/doc/example/CompleteNotificationDialogWithLongMessage.jsx

```diff
@@ -33,8 +33,8 @@ class CompleteNotificationDialogWithLongMessage extends React.Component {
           variant={NotificationDialogVariants.SUCCESS}
           isOpen={this.state.isOpen}
           onRequestClose={this.handleCloseModal}
-          title="This is the titleThis is the titleThis is the titleThis is the titleThis is the titleThis is the titleThis is the titleThis is the title"
-          message="This is the messageThis is the messageThis is the messageThis is the messageThis is the messageThis is the messageThis is the messageThis is the messageThis is the messageThis is the message"
+          title="This is the title. This is the title. This is the title. This is the title. This is the title. This is the title. This is the title. This is the title."
+          message="This is the message. This is the message. This is the message. This is the message. This is the message. This is the message. This is the message. This is the message. This is the message. This is the message."
```

Comment:
It might be worth looking into a more realistic title / message example we could substitute for this phrase while we're updating it. I think it would help our examples demo a lot better. If we can't find a long enough phrase I think even the lorem ipsum text would be an improvement.